com.unity.render-pipelines.core/CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [11.0.0] - 2020-10-21
+## [2022.2.0] - 2021-11-1
```

Comment:
Please add a `0` when it's a single-digit date, we had issues with validation tests in the past.
Example: `2021-11-01`

Reply:
Like in this test https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/srp-bumpversion/.yamato%252F_projectcontext.yml%2523test_all_project_windows_trunk/4831948/job

Reply:
Note: For all packages

Reply:
good point